cypress/e2e/audio-player/audio-player.spec.ts

```diff
@@ -0,0 +1,563 @@
+        const takeSnapshotTimeline = (layout: string) => {
+            cy.get(".mx_MainSplit .mx_RoomView_body").within(() => {
+                // Scroll to the bottom to make the audio player visible for Percy tests
+                cy.get(".mx_ScrollPanel").scrollTo("bottom");
+
+                // Click the timestamp to highlight the event tile on which player is rendered
+                cy.get(".mx_EventTile_last .mx_MessageTimestamp").click();
+                cy.get(".mx_EventTile.mx_EventTile_selected .mx_MAudioBody").should("be.visible");
+            });
+
+            cy.get(".mx_MainSplit").percySnapshotElement(`"Narrow main timeline with ThreadView on ${layout} layout"`, {
+                percyCSS,
+                widths: [600], // magic number
+            });
+            cy.log(`"Took a snapshot of Narrow main timeline with ThreadView on ${layout} layout"`);
+        };
+
+        visitRoom();
+
+        // Upload one second audio file with a long file name
+        uploadFile("cypress/fixtures/1sec-long-name-audio-file.ogg");
+
+        cy.get(".mx_RoomView_body .mx_RoomView_MessageList").within(() => {
+            // Assert the audio player is rendered
+            cy.get(".mx_EventTile_last .mx_AudioPlayer_container").should("exist");
+
+            // Click the timestamp to highlight the event tile on which player is rendered
+            cy.get(".mx_EventTile_last .mx_MessageTimestamp").click();
+            cy.get(".mx_EventTile.mx_EventTile_selected .mx_MAudioBody").should("be.visible");
+
+            cy.get(".mx_EventTile_last")
+                .realHover()
+                .within(() => {
+                    // Click "Reply in thread" button on MessageActionBar
+                    cy.get('[aria-label="Reply in thread"]').click({ force: false });
+                });
+        });
+
+        // Assert that the thread is visible
+        cy.get(".mx_ThreadView").should("be.visible");
+
+        // Take a snapshot of narrow main timeline with ThreadPanel opened on group layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+        takeSnapshotTimeline("group");
+
+        // Take a snapshot of narrow main timeline with ThreadPanel opened on bubble layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Bubble);
+        takeSnapshotTimeline("bubble");
+
+        // Reset to the default layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+
+        cy.get(".mx_ThreadView").within(() => {
+            // Click the timestamp to highlight the event tile on which player is rendered
+            cy.get(".mx_EventTile_last .mx_MessageTimestamp").click();
+            cy.get(".mx_EventTile.mx_EventTile_selected .mx_MAudioBody").should("be.visible");
+        });
+
+        // Take snapshots of audio player on a thread
+        takeSnapshots(".mx_ThreadView ", "Audio player on a thread");
```

Comment:
Do you think it would make the API of `takeSnapshots` easier to use if it was responsible for adding the space after the `wrapper`?

Reply:
Thanks for the suggestion, but this became obsolete since `wrapper` was removed with https://github.com/matrix-org/matrix-react-sdk/commit/82e4804aeb9d78544d5efe990a4d90b535a78090.